### ISSUE TITLE
[Adjust Rule] SIM113 should not trigger when adding a non-number

### ISSUE THREAD

**Issue by akx** [MartinThoma/flake8-simplify#55]
## Desired change

* **Rule(s)**: SIM113
* **Adjustment**: The rule should see whether the right side of the increment is a number.

## Explanation

Just looking at whether a variable is being incremented in a loop isn't enough to consider `enumerate()`.

## Example

This triggers `SIM113 Use enumerate instead of 'line'`:

```python
for line in read_list(redis_conn, storage_key):
    line += b'\n'
    # ....
```

**Comment by MartinThoma** [comment]
Thank you for reporting this! This seems easy to fix; I'll probably do it tomorrow evening :-)
